README.md

```diff
@@ -7,7 +7,7 @@
   <a href="https://circleci.com/gh/ResultadosDigitais/matrix"><img alt="CircleCI Status" src="https://img.shields.io/circleci/project/github/babel/babel/master.svg?label=CircleCI&maxAge=43200"></a>
 </p>
 
-## Welcome to the **#matrix**
+# Welcome to **#matrix**
```

Comment:
There's "the", an example: https://www.youtube.com/watch?v=QdurBIvrIWg

And the ideal even if it is two "#" because the main topic is the first line